plugins/src/webrtcsink/imp.rs

```diff
@@ -242,22 +245,45 @@ impl Default for State {
     }
 }
 
+fn make_converter_for_video_caps(caps: &gst::Caps) -> Result<gst::Element, Error> {
+    assert!(caps.size() == 1);
+
+    for feature in caps.features(0) {
+        if feature.contains(CUDA_MEMORY_FEATURE) {
+            return Ok(gst::parse_bin_from_description(
+                "cudaupload ! cudaconvert ! cudascale ! videorate drop-only=true",
+                true,
+            )?
+            .upcast());
+        } else if feature.contains(GL_MEMORY_FEATURE) {
+            return Ok(gst::parse_bin_from_description(
+                "glupload ! glcolorconvert ! glcolorscale ! videorate drop-only=true",
```

Comment:
```suggestion
                "glupload ! glcolorconvert ! glcolorscale ! videorate drop-only=true skip-to-first=true",
```

Reply:
interesting, I haven't checked the code but I would have thought `drop-only` would imply that?

Reply:
That could make sense but I don't think it does that

Reply:
OK, can't hurt to add anyway, it matches the intention and I can't tell from a quick read of the source code whether `drop-only` alone results in the same behaviour